packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -216,15 +228,22 @@ export const ChannelView: React.FC = () => {
           />
         </PaginationContainer>
       </>
+    ) : currentTab === 'NFTs' ? (
+      <VideoSection className={transitions.names.slide}>
+        {isSearching && <EmptyFallback title={`No videos matching "${searchQuery}" query found`} variant="small" />}
```

Comment:
Copy here is talking about videos which is wrong

Reply:
This come from the design https://www.figma.com/file/POw4j95jszfIP5bGfas2IC/Channel-page?node-id=581%3A43083

What copy should we have here?

Reply:
I guess "no NFTs"..., @toiletgranny ?

Reply:
I've added your suggestion, sounds reasonable. But let's wait for @toiletgranny

Reply:
Hey guys, sorry for my late response, I've been off last week. You're correct — the copy in designs was wrong. I've changed it to be `No NFTs matching “Border Collie” query found`.